arch/arm/core/swap_helper.S

```diff
@@ -80,8 +80,24 @@ SECTION_FUNC(TEXT, __pendsv)
 #elif defined(CONFIG_ARMV7_M_ARMV8_M_MAINLINE)
     stmia r0, {v1-v8, ip}
 #ifdef CONFIG_FP_SHARING
+    /* Assess whether switched-out thread had been using the FP registers. */
```

Comment:
Check maybe? Assess seems like we cannot be 100% sure about it.

Reply:
Thanks, that's not applicable any more (the original PR containing the patch is merged)
I'll track your suggestion in a separate issue.